assessment.md

```diff
@@ -36,11 +36,34 @@ Project Management is Covered by Team 2
 * Using the GitHub Flow model
 * Using the GitHub issue tracker
 * Assigning team members to issues in the GitHub issue tracker
+  * N = No team member was assigned to issue.
+  * I = Team member was assigned to issue but no response was given.
+  * A = Team member was assigned to issue and team member tried to assist
+  but no change was made.
+  * G = Team member was assigned to issue and issue was resolved.
+  * E = Team member was assigned to issue and issue was resolved and understood.
 * Creating and assigning labels to issues in the GitHub issue tracker
 * Creating and using a repository branch
 * Creating and using a repository fork
 * Merging a branch or a fork to another branch or fork
+  * N = Not making a branch when it your job
+  * I = when merging a branch or a fork to another branch or fork does not work
+  * A = when merging a branch or a fork to another branch or fork works only
+  sometimes
+  * G = when merging a branch or a fork to another branch or fork works most of
+  the time
+  * E = when merging a branch or a fork to another branch or fork works all the
+  time
```

Comment:
I recommend rewording these. I also suggest making a standard format with capitalizing after the equal sign or not

Reply:
It should be capitalized in this case.